scripts/west_commands/runners/nrfjprog.py

```diff
@@ -42,11 +48,15 @@ def do_add_parser(cls, parser):
                             help='if given, mass erase flash before loading')
         parser.add_argument('--snr', required=False,
                             help='serial number of board to use')
+        parser.add_argument('--tool-opt', default=[], action='append',
+                            help='''Additional options for nrfjprog,
+                    e.g. \'-coprocessor\' ''')
```

Comment:
```suggestion
                            e.g. "--recover"''')
```

Reply:
This suggestion combines a whitespace nit with a suggested change that applies to more families.

Reply:
Sure, I'll fix that.